homework9/task02/task02.py

```diff
@@ -0,0 +1,26 @@
+"""
+Write a context manager, that suppresses passed exception.
+Do it both ways: as a class and as a generator.
+>> with supressor(IndexError):
+...    [][2]
+"""
+from contextlib import contextmanager
+
+
+class Suppressor:
+    def __init__(self, name_exception):
+        pass
+
+    def __enter__(self):
+        pass
+
+    def __exit__(self, exc_type, exc_val, exc_tb):

```

Comment:
your class will supress every exception here. You will need to change it to catch only the ones you specify